src/ShellCheck/Checks/ShellSupport.hs

```diff
@@ -306,7 +308,8 @@ checkBashisms = ForShell [Sh, Dash] $ \t -> do
             ("trap", Just []),
             ("type", Just []),
             ("ulimit", if isDash then Nothing else Just ["f"]),
-            ("umask", Just ["S"])
+            ("umask", Just ["S"]),
+            ("unset", ["f", "v"])
```

Comment:
I updated this part to allow not checking flags (because `ulimit`'s list for Dash was getting really long and it was missing some options, so it seemed better not try to check in that case).

All that's needed is a `Just` in front of `["f", "v"]`

Reply:
I had somehow forgotten to commit the change, so it compiled and passed tests locally, but I pushed an outdated commit. Sorry for that.